<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Room</class>
 <widget class="QWidget" name="Room">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>直播间</string>
  </property>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>1104</x>
     <y>667</y>
     <width>100</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>70</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { 
	background-color: rgb(80, 236, 136,200);
	border-style:outset; 
	border-width:4px;
	border-radius: 10px;
	border-color:rgba(255,255,255);
	color: rgb(255, 255, 255); 
} 
QPushButton:hover { 
	background-color: rgb(80, 236, 136); 
	border-color:rgba(255,255,255);
}
QPushButton:pressed { 
	background-color: rgb(71, 215, 47); 
	border-color:rgba(255,255,255);
}</string>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>90</y>
     <width>189</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="text">
    <string>聊天室成员列表</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>824</x>
     <y>139</y>
     <width>380</width>
     <height>405</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>380</width>
     <height>405</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>1099</x>
     <y>94</y>
     <width>100</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>75</width>
     <height>23</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { 
	background-color: rgb(80, 236, 136,200);
	border-style:outset; 
	border-width:4px;
	border-radius: 10px;
	border-color:rgba(255,255,255);
	color: rgb(255, 255, 255); 
} 
QPushButton:hover { 
	background-color: rgb(80, 236, 136); 
	border-color:rgba(255,255,255);
}
QPushButton:pressed { 
	background-color: rgb(71, 215, 47); 
	border-color:rgba(255,255,255);
}</string>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>79</x>
     <y>569</y>
     <width>1125</width>
     <height>90</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>79</x>
     <y>139</y>
     <width>720</width>
     <height>405</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>720</width>
     <height>405</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>720</width>
     <height>405</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>79</x>
     <y>674</y>
     <width>1011</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>40</y>
     <width>721</width>
     <height>76</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>25</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white</string>
      </property>
      <property name="text">
       <string>主播的直播间</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>13</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>15</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:white</string>
          </property>
          <property name="text">
           <string>开播</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>35</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>15</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton { 
	background-color: rgb(161, 132, 255,200);
	border-style:outset; 
	border-width:4px;
	border-radius: 10px;
	border-color:rgba(255,255,255);
	color: rgb(255, 255, 255); 
} 
QPushButton:hover { 
	background-color: rgb(161, 132, 255); 
	border-color:rgba(255,255,255);
}
QPushButton:pressed { 
	background-color: rgb(110, 69, 245); 
	border-color:rgba(255,255,255);
}
QPushButton:!enabled { 
	background-color: rgb(100, 100,100,150); 
	border-color:rgba(255,255,255);
}</string>
          </property>
          <property name="text">
           <string>开</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>35</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>15</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton { 
	background-color: rgb(161, 132, 255,200);
	border-style:outset; 
	border-width:4px;
	border-radius: 10px;
	border-color:rgba(255,255,255);
	color: rgb(255, 255, 255); 
} 
QPushButton:hover { 
	background-color: rgb(161, 132, 255); 
	border-color:rgba(255,255,255);
}
QPushButton:pressed { 
	background-color: rgb(110, 69, 245); 
	border-color:rgba(255,255,255);
}
QPushButton:!enabled { 
	background-color: rgb(100, 100,100,150); 
	border-color:rgba(255,255,255);
}</string>
          </property>
          <property name="text">
           <string>关</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>13</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>15</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:white</string>
          </property>
          <property name="text">
           <string>摄像头</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="pushButton_7">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>35</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>15</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton { 
	background-color: rgb(161, 132, 255,200);
	border-style:outset; 
	border-width:4px;
	border-radius: 10px;
	border-color:rgba(255,255,255);
	color: rgb(255, 255, 255); 
} 
QPushButton:hover { 
	background-color: rgb(161, 132, 255); 
	border-color:rgba(255,255,255);
}
QPushButton:pressed { 
	background-color: rgb(110, 69, 245); 
	border-color:rgba(255,255,255);
}
QPushButton:!enabled { 
	background-color: rgb(100, 100,100,150); 
	border-color:rgba(255,255,255);
}</string>
          </property>
          <property name="text">
           <string>开</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_8">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>35</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>15</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton { 
	background-color: rgb(161, 132, 255,200);
	border-style:outset; 
	border-width:4px;
	border-radius: 10px;
	border-color:rgba(255,255,255);
	color: rgb(255, 255, 255); 
} 
QPushButton:hover { 
	background-color: rgb(161, 132, 255); 
	border-color:rgba(255,255,255);
}
QPushButton:pressed { 
	background-color: rgb(110, 69, 245); 
	border-color:rgba(255,255,255);
}
QPushButton:!enabled { 
	background-color: rgb(100, 100,100,150); 
	border-color:rgba(255,255,255);
}</string>
          </property>
          <property name="text">
           <string>关</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_7">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>17</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>15</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:white</string>
          </property>
          <property name="text">
           <string>麦克风</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>35</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>15</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton { 
	background-color: rgb(161, 132, 255,200);
	border-style:outset; 
	border-width:4px;
	border-radius: 10px;
	border-color:rgba(255,255,255);
	color: rgb(255, 255, 255); 
} 
QPushButton:hover { 
	background-color: rgb(161, 132, 255); 
	border-color:rgba(255,255,255);
}
QPushButton:pressed { 
	background-color: rgb(110, 69, 245); 
	border-color:rgba(255,255,255);
}
QPushButton:!enabled { 
	background-color: rgb(100, 100,100,150); 
	border-color:rgba(255,255,255);
}</string>
          </property>
          <property name="text">
           <string>开</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_4">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>35</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>15</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton { 
	background-color: rgb(161, 132, 255,200);
	border-style:outset; 
	border-width:4px;
	border-radius: 10px;
	border-color:rgba(255,255,255);
	color: rgb(255, 255, 255); 
} 
QPushButton:hover { 
	background-color: rgb(161, 132, 255); 
	border-color:rgba(255,255,255);
}
QPushButton:pressed { 
	background-color: rgb(110, 69, 245); 
	border-color:rgba(255,255,255);
}
QPushButton:!enabled { 
	background-color: rgb(100, 100,100,150); 
	border-color:rgba(255,255,255);
}</string>
          </property>
          <property name="text">
           <string>关</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
